specs/p2p.md

```diff
@@ -0,0 +1,17 @@
+# Peer To Peer (p2p) specifications
+## Introduction
+
+BCDBs should be able to communicate with each other to share and validate data. It means a BCDB server need to impersonate its user and be able to use his identity. A bcdb server then can create requests to other BCDB instance that are signed correctly. The receiving BCDB instance then can verify the identity
+of the caller by verifying the user signature against his public key published on the explorer. Once identity of the caller is verified, the receiver bcdb
+then can consult its local ACLs to see if the call is granted or not.
+
+While most of this logic is already in place this raises other questions:
+- How a BCDB instance find the public address of another instance. - Different ACLs that can be granted to users.
```

Comment:
The phonebook is the answer for this. 
While this is an easy answer, it indeed raises a lot of concerned regarding decentralization and single point of failure in the system.